packages/view3d-optimizer/index.js

```diff
@@ -33,93 +32,94 @@ program.parse(process.argv);
 
 const options = program.opts();
 
-tmp.withDir(async tempDir => {
-  const modelName = path.basename(options.input, path.extname(options.input));
-  const cwd = process.cwd();
-  const tmpFile = `${tempDir.path}/${modelName}.gltf`;
-  const outDir = options.output
-    ? path.resolve(cwd, options.output)
-    : path.resolve(path.dirname(options.input), modelName);
-
-  const run = async (cmd, taskDesc) => {
-    if (!options.silent) {
-      process.stdout.write(`- ${chalk.yellow(taskDesc)}...`);
-    }
-
-    const startTime = performance.now();
-    const tasks = (Array.isArray(cmd) ? cmd : [cmd]).map(task => exec(task));
-    await Promise.all(tasks).catch(err => {
-      process.stdout.write("\n");
-      err.stdout && console.error(chalk.red(err.stdout));
-      err.stderr && console.error(chalk.red(err.stderr));
-      process.exit(1);
-    });
-    const endTime = performance.now();
+const modelName = path.basename(options.input, path.extname(options.input));
+const cwd = process.cwd();
+const tmpFile = path.resolve(cwd, "temp.glb");
```

Comment:
Is this PR related to tmp?


The tmp-promise is missing from the code, but it seems to be in the package.json.


The module specified the tmp file directly, and is there any way to remove it again?

Reply:
You're right, I was just about to do that with the other fix :)